app/forms/reactivate_profile_form.rb

```diff
@@ -68,6 +68,12 @@ def valid_password?
     user.valid_password?(password)
   end
 
+  def recovery_code_decrypts?
```

Comment:
Ok, for some reason I thought that `RecoveryCodeGenerator#verify` actually tried decrypting, but it just compares the recovery code to its hash on the profile. This makes more sense now